assets/js/modules/tagmanager/datastore/__factories__/builders.js

```diff
@@ -146,7 +148,8 @@ export const defaultTagWeb = ( { accountId, containerId } = {} ) => ( {
 	blockingRuleId: null,
 	blockingTriggerId: null,
 	containerId,
-	fingerprint: Date.now().toString(),
+	fingerprint:
+		'Wed Feb 15 2023 19:01:10 GMT+0000 (Western European Standard Time)',
```

Comment:
This field is an integer as a string, e.g. https://github.com/google/site-kit-wp/blob/2094ab92472af431d7f24b804f78bc6afc7989a2/assets/js/modules/tagmanager/datastore/__fixtures__/get-containers--web.json#L11

Please use this format for consistency.

Reply:
Oh shoot, of course. I forget that `Date.now()` returns a number. 👍🏻